app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -2,7 +2,9 @@ module Authentication
   module AuthnJwt
     # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
     class ConfigurationJWTGenericVendor < ConfigurationInterface
+
       def initialize
+        @token_fetcher = Authentication::AuthnJwt::FetchTokenFromCredentials.new
```

Comment:
please call this param in the same name as the class it represents (`fetch_token_from_credentials`)

Reply:
The change is related to previous PR.